src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -67,10 +80,17 @@
      */
     public ViewClassificationAttributes(final Composite parent,
                                     final Controller controller) {
+        
+        // Load images
+        IMAGE_INSENSITIVE       = controller.getResources().getManagedImage("bullet_green.png"); //$NON-NLS-1$
+        IMAGE_SENSITIVE         = controller.getResources().getManagedImage("bullet_purple.png"); //$NON-NLS-1$
+        IMAGE_QUASI_IDENTIFYING = controller.getResources().getManagedImage("bullet_yellow.png"); //$NON-NLS-1$
+        IMAGE_IDENTIFYING       = controller.getResources().getManagedImage("bullet_red.png"); //$NON-NLS-1$

```

Comment:
It would probably a good idea to move this to the class resources and implement a method getImage(AttributeType).